documentation/chatmeleon-swagger.yaml

```diff
@@ -151,22 +175,15 @@ paths:
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  allMessages:
-                    type: array
-                    items:
-                      $ref: "#/components/schemas/Message"
-                  nextCursor:
-                    type: string
-                    pattern: "^[0-9a-f]{24}$"
-                    description: The ID of the earliest message of the current messages page.
+                type: array
+                items:
+                  type: object
+                  $ref: "#/components/schemas/Message"
+
         "401":
           description: Unauthorized - JWT token is missing or invalid.
         "403":
-          description: Forbidden - User not allowed to retrieve the messages of the conversation.
-        "404":
-          description: Not Found - Conversation not found.
+          description: Unauthorized - User is unauthorized.
   /messages:
     post:
       description: Create a new message.
```

Comment:
Change this route path to `POST /conversations/{conversationId}/messages` to be consistent with earlier GET route. And of course, remove the conversationId field in the request body down below.